index.html

```diff
@@ -1,11 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
+
 <body>
-    
+    <h1>Daniel Beggs</h1>
+    <div>
+        <ul class="nav">
```

Comment:
nice nice, awesome job with the navigation and the "About", "projects" "contact" etc and linking them to their proper pages!